Domain.Sql/SqlAzureDatabaseProperties.cs

```diff
@@ -0,0 +1,22 @@
+namespace Microsoft.Its.Domain.Sql
+{
+    /// <summary>
+    /// Sql Azure Database Properties
+    /// </summary>
+    public class SqlAzureDatabaseProperties
+    {
+        /// <summary>
+        /// Edition
+        /// </summary>
+        public string Edition { get; set; }
```

Comment:
We typically avoid setters in favor of a constructor, with appropriate validation, and readonly properties.